statsmodels/tsa/vecm/vecm.py

```diff
@@ -0,0 +1,2146 @@
+    stderr_alpha :  ndarray (neqs x r)
+        The standard errors of :math:`\\alpha`.
+    stderr_beta : ndarray (neqs x r)
+        The standard errors of :math:`\\beta`.
+    stderr_det_coef_coint : ndarray (num_det_coef_coint x r)
+        The standard errors of estimated the parameters related to
+        deterministic terms inside the cointegration relation.
+    stderr_gamma : ndarray (neqs x neqs*(k_ar-1))
+        The standard errors of :math:`\\Gamma`.
+    stderr_det_coef : ndarray (neqs x det. terms outside the coint. relation)
+        The standard errors of estimated the parameters related to
+        deterministic terms outside the cointegration relation.
+    tvalues_alpha : ndarray (neqs x r)
+    tvalues_beta : ndarray (neqs x r)
+    tvalues_det_coef_coint : ndarray (num_det_coef_coint x r)
+    tvalues_gamma : ndarray (neqs x neqs*(k_ar-1))
+    tvalues_det_coef : ndarray (neqs x det. terms outside the coint. relation)
+    pvalues_alpha : ndarray (neqs x r)
+    pvalues_beta : ndarray (neqs x r)
+    pvalues_det_coef_coint : ndarray (num_det_coef_coint x r)
+    pvalues_gamma : ndarray (neqs x neqs*(k_ar-1))
+    pvalues_det_coef : ndarray (neqs x det. terms outside the coint. relation)
+    var_rep : (k_ar x neqs x neqs)
+        KxK parameter matrices A_i of the corresponding VAR representation. If
+        the return value is assigned to a variable A, these matrices can be
+        accessed via A[i] for i=0, ..., k_ar-1.
+    cov_var_repr : ndarray (neqs**2 * k_ar x neqs**2 * k_ar)
+        This matrix is called :math:`\\Sigma^{co}_{\\alpha}` on p. 289 in [1]_.
+        It is needed e.g. for impulse-response-analysis.
+    fittedvalues : ndarray (nobs x neqs)
+        The predicted in-sample values of the models' endogenous variables.
+    resid : ndarray (nobs x neqs)
+        The residuals.
+
+    References
+    ----------
+    .. [1] Lutkepohl, H. 2005. *New Introduction to Multiple Time Series Analysis*. Springer.
+    """
+
+    def __init__(self, endog_tot, exog, exog_coint, k_ar,
+                 coint_rank, alpha, beta, gamma, sigma_u, deterministic='nc',
+                 seasons=0, first_season=0, delta_y_1_T=None, y_min1=None,
+                 delta_x=None, model=None, names=None, dates=None):
+        self.model = model
+        self.y_all = endog_tot
+        self.exog = exog
+        self.exog_coint = exog_coint
+        self.names = names
+        self.dates = dates
+        self.neqs = endog_tot.shape[0]
+        self.k_ar = k_ar
+        self.deterministic = deterministic
+        self.seasons = seasons
+        self.first_season = first_season
+
+        self.r = coint_rank
+        if alpha.dtype == np.complex128 and np.all(np.imag(alpha) == 0):
+            alpha = np.real(alpha)
+        if beta.dtype == np.complex128 and np.all(np.imag(beta) == 0):
+            beta = np.real(beta)
+        if gamma.dtype == np.complex128 and np.all(np.imag(gamma) == 0):
+            gamma = np.real(gamma)
```

Comment:
For some combinations of deterministic terms Gamma was returned with `dtype=complex` although the imaginary part was zero. (The dtype was changed by `np.inv()` somewhere in the code.) I have edited the code to incorporate your hint with `real_if_close`. Thanks for that!